Documentation/operations/troubleshooting_clustermesh.rst

```diff
@@ -27,10 +27,47 @@ Generic
 Manual Verification of Setup
 ----------------------------
 
- #. Validate that the ClusterMesh subsystem is initialized by looking for a
-    ``cilium-agent`` log message like this::
+ #. Validate that each cluster is assigned a **unique** human-readable name as well
+    as a numeric cluster ID (1-255).
 
-       level=info msg="Initializing ClusterMesh routing" path=/var/lib/cilium/clustermesh/ subsys=daemon
+ #. Validate that the ClusterMesh apiserver is initialized correctly for each cluster
+
+    .. code-block:: shell-session
+
+        $ kubectl logs -n kube-system deployment/clustermesh-apiserver -c apiserver
+        ...
+        level=info msg="Connecting to etcd server..." config=/var/lib/cilium/etcd-config.yaml endpoints="[https://127.0.0.1:2379]" subsys=kvstore
+        level=info msg="Got lease ID 7c0281854b945c05" subsys=kvstore
+        level=info msg="Got lock lease ID 7c0281854b945c07" subsys=kvstore
+        level=info msg="Initial etcd session established" config=/var/lib/cilium/etcd-config.yaml endpoints="[https://127.0.0.1:2379]" subsys=kvstore
+        level=info msg="Successfully verified version of etcd endpoint" config=/var/lib/cilium/etcd-config.yaml endpoints="[https://127.0.0.1:2379]" etcdEndpoint="https://127.0.0.1:2379" subsys=kvstore version=3.4.13
+
+ #. Validate that the ClusterMesh is healthy with ``cilium status``
+
+    .. code-block:: shell-session
+
+        ClusterMesh:   1/1 clusters ready, 10 global-services
+           k8s-c2: ready, 3 nodes, 8 identities, 10 services, 0 failures (last: never)
+           └  etcd: 1/1 connected, lease-ID=7c028201b53de660, lock lease-ID=7c028201b53de662, has-quorum=true: https://k8s-c2.mesh.cilium.io:2379 - 3.4.13 (Leader)
```

Comment:
We aren't benefiting from the shell-session highlighting here either, given that there's no shell characters. `::` is fine to use here.